Frontend/src/app/shared/components/navbar/navbar.component.html

```diff
@@ -1,19 +1,73 @@
-<div id="desktop">
-  <nb-tabset fullWidth class="nav" (changeTab)="changeTab($event)">
-    <nb-tab tabTitle="Time"> </nb-tab>
-    <nb-tab tabTitle="Projects"> </nb-tab>
-    <nb-tab tabTitle="Reports"> </nb-tab>
-    <nb-tab tabTitle="Manage"> </nb-tab>
-    <nb-tab tabTitle="Help"> </nb-tab>
-    <nb-tab tabTitle="Name" [tabIcon]="!isMenuOpen ? 'arrow-down' : 'arrow-up'">
-      <nb-menu [items]="deskItems" *ngIf="isMenuOpen"></nb-menu>
-    </nb-tab>
-  </nb-tabset>
-</div>
+<div class="desktop">
+  <div id="in-app-nav-desk" *ngIf="!hasRoute('/home')">
+    <nb-tabset fullWidth class="nav" (changeTab)="changeTab($event)">
+      <nb-tab tabTitle="Time"> </nb-tab>
+      <nb-tab tabTitle="Projects"> </nb-tab>
+      <nb-tab tabTitle="Reports"> </nb-tab>
+      <nb-tab tabTitle="Manage"> </nb-tab>
+      <nb-tab tabTitle="Help"> </nb-tab>
+      <nb-tab
+        tabTitle="Name"
+        [tabIcon]="!isMenuOpen ? 'arrow-down' : 'arrow-up'"
+      >
+        <nb-menu [items]="deskItems" *ngIf="isMenuOpen"></nb-menu>
+      </nb-tab>
+    </nb-tabset>
+  </div>
 
-<div id="mobile">
-  <nb-menu [items]="mobItems" autoCollapse="true" class="nav">
-  </nb-menu>
+  <div id="home-nav-desk" *ngIf="hasRoute('/home')">
+    <a (click)="homeClick()"> <img src="assets/logo.png" alt="logo" /> </a>
+    <div class="all-link-section">
+      <div class="left-link-section">
+        <p><a (click)="featuresClick()">Features</a></p>
+        <p><a (click)="integrationClick()">Integration</a></p>
+      </div>
+      <div class="right-link-section">
+        <p><a (click)="loginClick()">Login</a></p>
+        <button nbButton><a (click)="signupClick()">Sign up</a></button>
+      </div>
+    </div>
+  </div>
```

Comment:
I think this can be fixed in the navbar story but there shouldn't be 2 different navbars in that case we would just want 2 different components. But certain links should show or hide based on the page.

Reply:
Noted to be fixed in navbar.